tests/test_methods.py

```diff
@@ -103,16 +103,14 @@ def test_sgpr_qu():
 
     @tf.function
     def closure():
-        return - model.log_likelihood()
+        return model.neg_log_marginal_likelihood()
 
     gpflow.optimizers.Scipy().minimize(closure, variables=model.trainable_variables)
+    [qu_m, qu_A] = model.compute_qu()
+    [f_atZ_m, f_atZ_A] = model.predict_f(model.inducing_variable.Z, full_cov=True)
```

Comment:
```suggestion
    f_at_Z_mean, f_at_Z_cov = model.predict_f(model.inducing_variable.Z, full_cov=True)
```
(or `f_Z_mean` or so if you want to be more concise)